src/WireMock.Net.AspNetCore.Middleware/HttpDelegatingHandler/IWireMockDelegationHandlerSettings.cs

```diff
@@ -0,0 +1,8 @@
+// Copyright © WireMock.Net
+
+namespace WireMock.Net.AspNetCore.Middleware.HttpDelegatingHandler;
+
+internal interface IWireMockDelegationHandlerSettings
```

Comment:
Why are we creating an interface for this?  This isn't really necessary as how you're registering it in the ServiceCollectionExtensions.cs class could just be done with a standard class (no interface required).  This would reduce the need for 1 extra file.

Reply:
Good point. I will take a look.

However can you checkout my branch and verify if this works? For me it does not work...